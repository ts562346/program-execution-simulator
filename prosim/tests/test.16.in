test1 8
LOOP 4
  BLOCK 2
  LOOP 3
    DOOP 1
  END
  BLOCK 5
END
HALT

