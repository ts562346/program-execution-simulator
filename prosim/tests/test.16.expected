00000: BLOCK
00002: DOOP
00003: DOOP
00004: DOOP
00005: BLOCK
00010: BLOCK
00012: DOOP
00013: DOOP
00014: DOOP
00015: BLOCK
00020: BLOCK
00022: DOOP
00023: DOOP
00024: DOOP
00025: BLOCK
00030: BLOCK
00032: DOOP
00033: DOOP
00034: DOOP
00035: BLOCK
00040: HALT

DOOP count : 12
DOOP time  : 12
BLOCK count: 8
BLOCK time : 28
